h5p-bildetema-words-topic-image/src/components/TopicImageWordAudio/TopicImageWordAudio.module.scss

```diff
@@ -0,0 +1,60 @@
+@use "../../../../common/abstracts" as *;
+
+.wordAudio {
+  padding: 0.25rem 0;
+
+  &.selected button {
+    outline: 3px solid $orange;
+  }
+}
+
+.wordAudio button {
+  all: unset;
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  gap: 0.25rem;
+  padding: 0.25rem 0.5rem;
+}
+
+.word_label {
+  font-size: $font-size-24;
+  line-height: 1.4;
+  text-align: center;
+}
+
+.audioIconSpan {
+  align-items: center;
+  display: flex;
+  height: 2.1rem; /* same as label height */
```

Comment:
We miiiight want to use `em` here, to make the height change if the font size changes.